src/utils/exportData.ts

```diff
@@ -29,40 +28,39 @@ export const MainRowKeys = {
   contactEmail: 'Overall Contact Email',
 };
 
-const getMainRow = (studyProps: StudyProps): Record<string, string> =>
+const getMainRow = (studyProps: StudyDetailProps): Record<string, string> =>
   convertToSpreadsheetRow([
     { header: MainRowKeys.trialId, body: studyProps.trialId },
-    { header: MainRowKeys.likelihood, body: studyProps.likelihood.text },
-    { header: MainRowKeys.title, body: studyProps.title },
-    { header: MainRowKeys.status, body: studyProps.status.text },
-    { header: MainRowKeys.period, body: studyProps.period },
-    { header: MainRowKeys.phase, body: studyProps.phase },
-    { header: MainRowKeys.conditions, body: JSON.stringify(studyProps.conditions) },
-    { header: MainRowKeys.type, body: studyProps.type },
-    { header: MainRowKeys.description, body: studyProps.description },
-    { header: MainRowKeys.eligibility, body: studyProps.eligibility },
-    { header: MainRowKeys.sponsor, body: studyProps.sponsor.name },
-    { header: MainRowKeys.contactName, body: studyProps.contacts?.[0]?.name },
-    { header: MainRowKeys.contactPhone, body: studyProps.contacts?.[0]?.phone },
-    { header: MainRowKeys.contactEmail, body: studyProps.contacts?.[0]?.email },
+    { header: MainRowKeys.likelihood, body: studyProps.likelihood?.text || '' },
+    { header: MainRowKeys.title, body: studyProps.title || '' },
+    { header: MainRowKeys.status, body: studyProps.status?.text || '' },
+    { header: MainRowKeys.period, body: studyProps.period || '' },
```

Comment:
Not related to this PR, but I figure the comment is relevant since this PR changes the spreadsheet rows. 
I noticed that the trials from BCT.org don't seem to pull in a `period`. I'm not sure if this is always the case for the other wrappers. 

For instance, here are the 136 trials from today, 2/11/2022: 
[clinicalTrials.xlsx](https://github.com/mcode/clinical-trial-matching-app/files/8050483/clinicalTrials.xlsx)
 
Putting a `console.log(study.period);` in `getPeriod` from `src/components/Results/utils.ts` shows all of the `ResearchStudy.period`s are undefined.

Reply:
So it seems that none of the results from the wrappers return a period. I wrote the `Bundle` of results to a json file and did a search for the following regular expressions:
* `[ "]?end[" ]?:`
* `[ "]?start[" ]?:`
* `"20\d{2}[^\d]`

I received no results back (that weren't postal codes) so I do not think the results from the wrappers have a date -- thus there's nothing we can really do here. That being said, outside of this PR, @zlister should we perhaps write a ticket to update the backup service to perhaps grab this information from CT.gov?

Reply:
Yes @ChristineDuong I will put in a ticket to pull that information from CT.gov. It will probably also be nice to display that info on the details card somewhere. Good catch.